packages/react/src/ActionList/List.tsx

```diff
@@ -33,15 +33,20 @@ export const List = React.forwardRef<HTMLUListElement, ActionListProps>(
 
     /** if list is inside a Menu, it will get a role from the Menu */
     const {
-      listRole,
+      listRole: listRoleFromContainer,
       listLabelledBy,
       selectionVariant: containerSelectionVariant, // TODO: Remove after DropdownMenu2 deprecation
-      enableFocusZone,
+      enableFocusZone: enableFocusZoneFromContainer,
     } = React.useContext(ActionListContainerContext)
 
     const ariaLabelledBy = slots.heading ? slots.heading.props.id ?? headingId : listLabelledBy
-
+    const listRole = role || listRoleFromContainer
     const listRef = useProvidedRefOrCreate(forwardedRef as React.RefObject<HTMLUListElement>)
+
+    let enableFocusZone = false
+    if (enableFocusZoneFromContainer !== undefined) enableFocusZone = enableFocusZoneFromContainer
+    else if (listRole) enableFocusZone = ['menu', 'menubar', 'listbox'].includes(listRole)
+
     useFocusZone({
```

Comment:
I'm wondering if we'd want to add `focusOutBehavior: 'wrap'` by default, or allow consumers to add that themselves?

Reply:
> I'm wondering if we'd want to add focusOutBehavior: 'wrap' by default

Honestly, not sure, what do you think?

> or allow consumers to add that themselves?

That's the part that's missing right now. I think I'd like to wait for feedback/requests before introducing `focusZoneSettings` like we have on other focus-zoning components

Reply:
For `role=menu`, we'd definitely want it to wrap by default. With `role=listbox` there may be cases where it doesn't need to wrap. I'd say if we can conditionally set it to `wrap` based on if it's a menu or not would be a safe bet, at least for now!

Reply:
Done!